browser/src/components/NodeSummary.tsx

```diff
@@ -108,26 +108,53 @@ export function NodeSearch({ graphManager, weightService, nodeName }: SearchProp
       </div>
     )
   }
-  // will partially display each nodes summary
   if (searchResult.length > 1) {
-    {searchResult.map(element => {
-      if(displayNameHelper.displayNameForKey(element.node.key) == nodeName){
-        //TODO: If nodeName is CoffeeMaker && results list contains CoffeeMakerController & CoffeeMaker return CoffeeMaker
-      }
-    })}
-    return (
-      <div>
-        {searchResult.map(element => {
-          var prop : Props = {graphManager, weightService, componentName : element.componentName, nodeName : element.node.key, fullDetails: false}
-          return NodeSummary(prop)
-        })}
-      </div>
-    )
-  }
+    //return fullNode Summary if exactly 1 of the result nodes are equal to the short name of a node 
```

Comment:
the method name `shortNameEqualsNodeName` doesn't match the comment `return fullNode Summary if exactly 1 of the result nodes are equal to the short name of a node `, maybe think a better name?